Library/Homebrew/utils.rb

```diff
@@ -24,9 +24,13 @@ def require?(path)
   raise unless e.message.include?(path)
 end
 
-def ohai(title, *sput)
+def ohai_title(title)
   title = Tty.truncate(title) if $stdout.tty? && !ARGV.verbose?
-  puts Formatter.headline(title, color: :blue)
+  Formatter.headline(title, color: :blue)
+end
```

Comment:
Not sure I understand this change. What's wrong with `oh1` for this case?

Reply:
I needed a function that would return the formatted string that `ohai` prints, instead of printing it immediately with `puts`.